### PR TITLE
homepage: partners

### PR DESCRIPTION
**Pull request by amanuela97**
# Why ?
As a visitor, i would like to know the partners of the company and the review of the company.
Lastly, As a user i want the footer to look nice so i can easily find the information i want. See [Trello card](https://trello.com/c/ssPUdRPF/385-reviews-and-our-partners-on-a-first-section-hero-section)

# Acceptance Criteria
- [x] hero section buttons updated
- [x] hero section background image removed
- [x] hero section includes review
- [x] partners section added
- [x] partners logo swipe effect/animation created
- [x] footer texts centered and #WEHELPUKRAINE removed

# Demo Link
- https://deploy-preview-62--wehelpvip.netlify.app/

### CODE REVIEW

File: components/block/partners.vue [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<template>
+  <div
+    class="flex flex-row items-center justify-center overflow-x-scroll sm:overflow-x-hidden no-scrollbar h-48 sm:items-end sm:h-24 gap-8 bg-white-500"
+  >
+    <div class="flex align-middle gap-8 h-8 sm: sm:h-2/5 px-5 sm:px-0 ">
+      <img src="/partners/pg-avogados.svg" alt="PG avogados" />
```
Comment (razbakov) [inline components/block/partners.vue:6]: Please define an array and add cycle to render it

File: components/block/partners.vue [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+      <img src="/partners/banovo.svg" alt="Banovo" />
+      <img src="/partners/snk.svg" alt="SNK" />
+    </div>
+  </div>
+</template>
+<style scoped>
+.no-scrollbar::-webkit-scrollbar {
```
Comment (razbakov) [inline components/block/partners.vue:13]: please add it in tailwind config

File: components/block/start.vue [hunk 3]
```diff
@@ -5,50 +5,72 @@
...
+          </div>
         </div>
       </div>
-      <div class="bg-no-repeat bg-top bg-favicon-transp h-60 w-full"></div>
     </main>
   </div>
 </template>
 <script>
 import { useI18n } from 'vue-i18n'
+import RatingItem from '../rating/item.vue'
```
Comment (razbakov) [inline components/block/start.vue:66]: it is autoimported, see nuxtjs docs

File: components/rating/item.vue [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    return {
+      stars: 5,
+    }
+  },
+  computed: {
+    unrated() {
+      return this.stars - parseInt(Math.round(`${this.rating}`))
+    },
+    rated() {
+      return parseInt(Math.round(this.rating))
```
Comment (razbakov) [inline components/rating/item.vue:54]: Math.round returns int, no need for parseInt

File: tailwind.config.js [hunk 5]
```diff
@@ -160,8 +160,8 @@ module.exports = {
             letterSpacing: '0px',
           },
         ],
-        body: [
-          '14px',
+        body1: [
```
Comment (razbakov) [inline tailwind.config.js:163]: body1?

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/service/item.vue [hunk 6]
```diff
@@ -71,12 +44,14 @@
 </template>
 <script>
 import { PhoneIcon, LinkIcon } from '@heroicons/vue/solid'
+import RatingItem from '../rating/item.vue'
```
Comment (razbakov) [inline components/service/item.vue:47]: all components in nuxt3 are auto-imported, see documentation

**Review by razbakov — CHANGES_REQUESTED** [review]

File: components/service/item.vue (hunk 6 above)
Comment (amanuela97) [reply components/service/item.vue:47]: yeah, i keep forgetting

**Review by razbakov — APPROVED** [review]

**Pull request merged**